res/controllers/common-hid-packet-parser.js

```diff
@@ -2077,9 +2077,9 @@ class HIDController {
                                 continue;
                             }
                             const bitControlGroup = this.resolveGroup(bit.mapped_group);
-                            engine.connectControl(
+                            engine.makeConnection(
                                 bitControlGroup, bit.mapped_name, bit.mapped_callback, true);
```

Comment:
makeConnection has not 4th parameter as connectControl. See https://github.com/mixxxdj/mixxx/blob/bdfb495e2458ecd1a8edd8b3bacd836654b76922/res/controllers/engine-api.d.ts#L107-L116
vs.
https://github.com/mixxxdj/mixxx/blob/bdfb495e2458ecd1a8edd8b3bacd836654b76922/res/controllers/engine-api.d.ts#L131-L143

Reply:
oh that's the difference! I was wondering